### PR TITLE
Prepare for 4.x release

### PR DESCRIPTION
**Pull request by Nyholm**
This updates travis.yml and will replace #48

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,11 +13,14 @@
         }
     ],
     "require": {
-        "php": "^7.0",
+        "php": "^7.1",
         "symfony/config": "^2.3 || ^3.0 || ^4.0"
```
Comment (xabbuh) [inline composer.json]: Time to go up to `^2.7 || ^3.4 || ^4.0` here?

Comment (Nyholm) [reply composer.json]: Yeah, fair.

File: composer.json [hunk 2]
```diff
@@ -13,11 +13,14 @@
         }
     ],
     "require": {
-        "php": "^7.0",
-        "symfony/config": "^2.3 || ^3.0 || ^4.0"
+        "php": "^7.1",
+        "symfony/config": "^2.7 || ^3.0 || ^4.0"
```
Comment (xabbuh) [inline composer.json]: Not `^3.4`?

Comment (Nyholm) [reply composer.json]: I read your comment to quickly.

**Comment by xabbuh** [comment]
👍

**Pull request merged**